webreloader.py

```diff
@@ -1,5 +1,15 @@
 import requests
 import time
+from pyfiglet import *
+
+def main():
+    f = Figlet()
```

Comment:
heding using pyfiglet 
slant font is used 
you may change as per your requirements.